pkg/pillar/containerd/oci.go

```diff
@@ -380,40 +381,10 @@ func (s *ociSpec) updateFromImageConfig(config v1.ImageConfig) error {
 // UpdateMounts adds volume specification mount points to the OCI runtime spec
 func (s *ociSpec) UpdateMounts(disks []types.DiskStatus) error {
 	ociVolumeData := "rootfs"
-	mountDirs := []string{}
 	blkMountPoints := ""
 
-	// We are prepared to deal with the first volume being declared as root,
-	// however any other volume claiming that right would be treated as it
-	// doesn't have MountDir specifies at all
-	id := 0
-	if len(disks) > 0 && disks[0].MountDir == "/" {
-		mountDirs = []string{"/"}
-		id = 1
-	}
-
-	// Validating if there are enough disks provided for the mount-points
-	if len(disks)-id < len(s.volumes) {
-		// If no. of mount-points is (strictly) greater than no. of disks provided, we need to throw an error as there
-		// won't be enough disks to satisfy required mount-points.
-		return fmt.Errorf("updateMounts: Number of volumes provided: %v is less than number of mount-points: %v",
```

Comment:
I'd rather convert this to warning then remove it altogether. BUT! A warning that only fires if `len(s.volumes) > 0`.

Reply:
I am not sure I agree on this. There need be absolutely no relationship between number of volumes in OCI config and in passed config. Because there need be absolutely no relationship between the volumes in OCI config and those in passed config.

containerd doesn't care; docker doesn't care; no runtime I can think of cares. Why would we warn? Doesn't that imply that there kind of / should of / sort of is a relationship, and you are sort of violating it?

Reply:
Well... I supposed that is a somewhat defensible argument. Lets go with it for now -- after all -- we can always add warnings back -- it is not like it would be a big change in behavior NOW.